tmva/tmva/inc/TMVA/DNN/Minimizers.h

```diff
@@ -76,8 +74,14 @@ class TGradientDescent
    TGradientDescent(Scalar_t learningRate,
                     size_t   convergenceSteps,
                     size_t   testInterval);
-   /** Reset minimizer object to initial state. Does nothing for this minimizer. */
-   void Reset() {};
+
+   /** Reset minimizer object to default state. */
+   void Reset()
+   {
+      fMinimumError = 1e100;
```

Comment:
Sure, I can use that if it's preferred. I wanted to make a minimal change in functionality compared to what was here before. I will update it.